pyogrio/tests/test_geopandas_io.py

```diff
@@ -1568,6 +1584,21 @@ def test_write_read_null(tmp_path, use_arrow):
     assert result_gdf["object_str"][2] is None
 
 
+def test_write_read_vsimem(naturalearth_lowres_vsi, use_arrow_write):
```

Comment:
Can you use `use_arrow` and then decorate with `@pytest.mark.requires_arrow_write_api`, which seems to be how the other tests are set up here? (then you don't need to define the new `use_arrow_write` fixture)

Reply:
OK, changed